libs/styles/styles.css

```diff
@@ -1066,26 +1068,24 @@ span.icon.margin-inline-end { margin-inline-end: var(--s2-spacing-100); }
 .con-button.button-xl span.icon.margin-left { margin-left: 14px; }
 
 .dark .con-button {
-  border-color: var(--color-white);
-  color: var(--color-white);
+  background-color: var(--s2-transparent-white-25);
+  color: var(--s2-transparent-white-900);
+  border-color: var(--s2-gray-800);
 }
 
-.light .con-button {
```

Comment:
Why do we need to completely remove the `.light` styles? It might be that they are still valuable for backwards compatibility across our pages.

Reply:
I can add these styles back in for backwards compatibility and remove the light properties from the blocks here instead. I initially had these removed because the spectrum designs were created to default to the `light` styles automatically, so it's only needed to toggle to `dark` when needed.